### PR TITLE
Redesign Product

### PR DESCRIPTION
**Pull request by brunoparma88**
Redesing product template (FE)
Add the route titles in products and details

### CODE REVIEW

**Review by enzofab91 — APPROVED** [review]

File: app/views/development_metrics/products/_general.erb [hunk 1]
```diff
@@ -1,97 +1,101 @@
-<div class="container">
-  <div class="summary">
-    <div class='chart-container <%if current_page?(products_development_metrics_path) && has_jira_project%>col-md-12 info'<% else %>'<% end %> >
-        <% if has_jira_project %>
-          <% if show_defect_escape_rate %>
+<div class="summary">
+  <div class='chart-container <%if current_page?(products_development_metrics_path) && has_jira_project%>col-md-12 info'<% else %>'<% end %> >
```
Comment (sebastiancaraballo) [inline app/views/development_metrics/products/_general.erb:2]: Is this okay?

File: app/views/development_metrics/products/_general.erb [hunk 2]
```diff
@@ -1,97 +1,101 @@
...
-                  <div class="box-title" ><%= development_cycle_definition && development_cycle_definition[:name] %></div>
-                  <span data-placement='right' class='metric_tooltip' style= "right:10px" aria-hidden='true' data-toggle='tooltip' title='<%= development_cycle_definition && development_cycle_definition[:explanation] %>'>!</span>
+      <% if show_development_cycle %>
+        <div class="col">
+          <div class="metrics" style="margin-right: 10px;">
+            <div class='row'>
+              <div class="box-title" ><%= development_cycle_definition && development_cycle_definition[:name] %></div>
+              <span data-placement='right' class='metric_tooltip' style= "right:10px" aria-hidden='true' data-toggle='tooltip' title='<%= development_cycle_definition && development_cycle_definition[:explanation] %>'>!</span>
+            </div>
+            <%if current_page?(products_metrics_development_metrics_path)%>
```
Comment (sebastiancaraballo) [inline app/views/development_metrics/products/_general.erb:42]:
```suggestion
            <% if current_page?(products_metrics_development_metrics_path) %>
```
Check space

File: app/views/development_metrics/products_metrics.erb [hunk 3]
```diff
@@ -1,40 +1,33 @@
...
+        defect_escape_rate: @defect_escape_rate,
+        development_cycle: @development_cycle,
+        planned_to_done: @planned_to_done,
+        development_cycle_definition: @development_cycle_definition,
+        defect_escape_rate_definition: @defect_escape_rate_definition,
+        planned_to_done_definition: @planned_to_done_definition,
+        has_jira_project: @has_jira_project,
+        show_defect_escape_rate: @show_defect_escape_rate,
+        show_development_cycle: @show_development_cycle,
+        show_planned_to_done: @show_planned_to_done %>
```
Comment (sebastiancaraballo) [inline app/views/development_metrics/products_metrics.erb:29]: Maybe it's a good idea to refactor this into one object in a future refactor 🧐

**Review by sebastiancaraballo — APPROVED** [review]
💯 Please remember to add screenshot when there are visual changes

**Pull request merged**